src/DynamoCoreWpf/Interfaces/IShellCom.cs

```diff
@@ -0,0 +1,502 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace Dynamo.Wpf.Interfaces
+{
+    internal static class IIDGuid
```

Comment:
I assume these are all standard stuff obtained from some documentation. Can you please include the link here as comment?